### ISSUE TITLE
Support bloom filters for ORC

### ISSUE THREAD

**Issue by electrum** [prestodb/presto#3071]
Hive 1.2.0 has bloom filters for ORC. This could be useful both for Hive and Raptor.

**Comment by yuananf** [comment]
Good feature! This can improve the predicate efficiency for this kind of query
```
select * from tpch_1000.lineitem where l_orderkey=1212000001;
```
Is Dain working on this new feature?

**Comment by dain** [comment]
Nope.

**Comment by raghuvineel** [comment]
May I know to which version this has been integrated to? Does presto.151 has bloom filters support for ORC tables?

**Comment by KBP-TDC** [comment]
@raghuvineel : this issue is Open, meaning it's not done yet.

**Comment by RobinUS2** [comment]
You will see the following error if you do try to use it from presto (this ran on 0.152)
```
BLOOM_FILTER stream type not implemented yet
```

**Comment by RobinUS2** [comment]
I'm exploring what needs to be done to implement this

**Comment by RobinUS2** [comment]
I will likely open a PR today that has implemented bloom filters for ORC files, if you want you can assign this to me to prevent confusion / double work.

**Comment by RobinUS2** [comment]
Fixed by #5998 , closing this one.
